setup.py

```diff
@@ -21,7 +21,7 @@
       requirements.append('enum34')
 
 setup(name='SQLAlchemy-JSONAPI',
-      version='4.0.9',
+      version='4.0.10',
```

Comment:
Let's remove this increment of the version string. We'll do that as part of rolling a new release, and I've also got a patch in bound to change how we set the version number this so we can get at the version programmatically.